cypress/integration/app/activateuser.spec.ts

```diff
@@ -0,0 +1,104 @@
+describe("Invite user", () => {
+  it("Enable SMTP setting", () => {
+    cy.setup();
+    cy.login();
+
+    cy.visit("/settings/organization");
+
+    cy.findByLabelText(/enable smtp/i).check({ force: true });
+
+    cy.findByLabelText(/sender address/i)
+      .click()
+      .type("rachel@fleetdm.com");
+
+    cy.findByLabelText(/smtp server/i)
+      .click()
+      .type("localhost");
+
+    cy.get("#port").click().type("{selectall}{backspace}1025");
+
+    cy.findByLabelText(/use ssl\/tls/i).check({ force: true });
+
+    cy.findByLabelText(/smtp username/i)
+      .click()
+      .type("rachelsusername");
+
+    cy.findByLabelText(/smtp password/i)
+      .click()
+      .type("rachelspassword");
+
+    cy.findByRole("button", { name: /update settings/i }).click();
+  });
+
+  it("Invite user", () => {
+    cy.findByRole("tab", { name: /^users$/i }).click();
+
+    cy.contains("button", /invite user/i).click();
+
+    cy.findByLabelText(/name/i).click().type("Ash Ketchum");
+
+    cy.findByLabelText(/email/i).click().type("ash@fleetdm.com");
+
+    cy.findByRole("button", { name: /invite/i }).click();
+
+    cy.wait(3000);
```

Comment:
Though `wait` is [listed as an anti-pattern](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting), I think it makes sense to use it here. Cypress isn't aware that we are waiting for the mail server to make the message available in this case.

I think it's reasonable to disable the lint check for this line:
```suggestion
    cy.wait(3000); // eslint-disable-line cypress/no-unnecessary-waiting
```

@ghernandez345 do you think otherwise?